evalml/pipelines/utils.py

```diff
@@ -119,6 +119,11 @@ def make_pipeline_from_components(component_instances, problem_type, custom_name
     Returns:
         Pipeline instance with component instances and specified estimator
 
+    Example:
+        >>> components = [Imputer(), StandardScaler(), Estimator()]
```

Comment:
I think you should say `RandomForestClassifier()` here, otherwise this example isn't runnable because `Estimator()` is abstract and will throw. Actually I thought we had docstrings get run during our docs, strange that this didn't fail!
```python
In [2]: evalml.pipelines.components.Estimator()
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-81c4f09910a6> in <module>
----> 1 evalml.pipelines.components.Estimator()

TypeError: Can't instantiate abstract class Estimator with abstract methods model_family, name, supported_problem_types
```

Reply:
Ah yeah this did fail here. I'm having an interesting problem with this end estimator in the docs. `Estimator` doesn't work for the reasons you stated, but linting keeps catching `RandomForestClassifier` as an "unused" import since it isn't used in the code itself. I tried `Catboost` but that would fail when there during the core dependencies check. 

I think the best solution for now is to import `RandomForestClassifier` and put in a `#noqa: F401` after this line.